src/middleware.ts

```diff
@@ -116,3 +119,34 @@ export const combine = <
       api as StoreApi<PrimaryState>
     )
   )
+
+export const persist = <S extends State>(
+  name: string,
+  config: StateCreator<S>,
```

Comment:
```suggestion
  config: StateCreator<S>,
  name: string = "zustand-persist-default-key",
```
How about this?

Reply:
If the user uses multiple persistent stores in the same app/website, it'll create some conflicts.

Reply:
Yeah. Do you prefer removing the default value?

Reply:
Yep, I think that forcing the user to name his store is better (at least if we don't find a solution to manage the names for him).